### DISCUSSION TITLE
Type of mouse?

### DISCUSSION THREAD

**Discussion by gerardvanschip** [dirkwhoffmann/vAmiga#808] [Q&A]
First off, what an awesome app. I used FS-UAE in the past, had not touched emulation for a few years and found this today, awesome. Was real quick to get it up and running and run some games but for whatever reason it does not seem to see my mouse. I use a logitech bluetooth mouse. Within vamiga I do see it saying "this device is not known to the emulator". Is there some trick to get your mouse workiung?

**Comment by dirkwhoffmann** [comment]
"this device is not known to the emulator" only means that vAmiga cannot find the device in its internal database. You might be lucky to still use it if it shows up in the popup menu marked below:

<img width="1004" alt="Bildschirmfoto 2024-01-06 um 12 24 56" src="https://github.com/dirkwhoffmann/vAmiga/assets/12561945/5ac5e3ed-4d07-462a-a5f6-75f0648ff66d">

It it doesn't work, you can't do much until you're a developer.
